lang/en.js

```diff
@@ -1,6 +1,7 @@
 export default {
 	"components.alert.close": "Close Alert",
 	"components.breadcrumbs.breadcrumb": "Breadcrumb",
+	"components.button-add.addItem": "Add Item",
```

Comment:
I only did `en` for now in case the component name changes.

Reply:
The linter isn't going to let you merge as-is. Typically we'll just use `mvf` to copy the English value everywhere.

Reply:
I thought I was going to manually have to update them so I was waiting to make sure the component name was good to go, but yay for automation! 🎉 Will fix